app/workers/clean_job.rb

```diff
@@ -0,0 +1,7 @@
+class CleanupWorker

```

Comment:
Consider having both jobs extend Rails' `ActiveJob::Base` rather than `include Sidekiq::Worker` so the jobs aren't coupled so closely to Sidekiq. 

By not having a tight coupling to Sidekiq, it would allow easier swapping in of alternative job backends and the ability to make use of [ActiveJob's test helper methods](http://api.rubyonrails.org/classes/ActiveJob/TestHelper.html).

Configuring ActiveJob for Sidekiq: http://guides.rubyonrails.org/active_job_basics.html#setting-the-backend